src/utils.zig

```diff
@@ -985,6 +985,12 @@ pub fn BlockingMaxHeap(comptime T: type) type {
 }
 
 pub const ThreadPool = struct {
+	pub const TaskType = enum(usize) {
+		chunkgen,
+		lighting,
+		misc,
+		taskTypes,
```

Comment:
Instead of having a C-style enum length field, I think it would be better to use `std.enums.directEnumArrayLen` to determine the length instead.
This would also make it possible to use `@tagName` above.

Reply:
Sometimes I wonder if Zig devs have a naming scheme.